CMakeLists.txt

```diff
@@ -105,6 +105,10 @@ if(LUABIND_NO_STD_SHARED_PTR)
     add_definitions(-DLUABIND_NO_STD_SHARED_PTR)
 endif()
 
+if(LUABIND_USE_CXX11)
```

Comment:
Yes, this needs to be in `build_information.hpp`. This PR is somehow live in Ubuntu 20.04 LTS though the ValyriaTear/luabind repo. It broke our ryzom/ryzomcore build, since it's built with the option, but the headers get included without LUABIND_USE_CXX11.

Reply:
The alternative is of course auto-detecting, and not giving the option to users.